docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -101,7 +101,7 @@ spec:
   deny:
     host_sudoers: [
        "*" # ensure that users in this role never have sudoers files created on matching Nodes
-       "%wheel ALL=(ALL) NOPASSWD: ALL" # host_sudoers entries matching this are filtered out
+       "ALL=(ALL) NOPASSWD: ALL" # host_sudoers entries matching this are filtered out
```

Comment:
Should there be an accompanying code change that filters out sudoers without prefixed username, or is that how it already works?